GoogleUtilities/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Unreleased
+- Update comments in `GULAppDelegateSwizzler` (#4518)
```

Comment:
Suggestion: these comment changes don't affect consumers of this package so don't bother your users with changelog entries. If users want to see everything we do they can browse git history. This should be a quick list of things that might affect them or might prove useful.

Reply:
Good point, I'll remove it then.